.scripts/ci.yml

```diff
@@ -53,6 +53,8 @@ stages:
               path: $(Build.ArtifactStagingDirectory)
       - job: HLC_Generation
         dependsOn: Build
+        variables:
```

Comment:
A better approach is to create a template global.yml with this variable and include that template in each job. You can refer similar implementation in js repo here.

global template in JS: `https://github.com/Azure/azure-sdk-for-js/blob/main/eng/pipelines/templates/variables/globals.yml`


And it's reference in job: https://github.com/Azure/azure-sdk-for-js/blob/main/eng/pipelines/templates/jobs/ci.yml#L31-L32